src/test/groovy/org/ods/doc/gen/AppSpec.groovy

```diff
@@ -12,6 +13,7 @@ import static org.hamcrest.Matchers.equalTo
 
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
 @ActiveProfiles("pro")
+@DirtiesContext
```

Comment:
Please add a comment to explain why this is needed.

Reply:
https://www.baeldung.com/spring-dirtiescontext